doc/mf6io/mf6ivar/dfn/gwf-uzf.dfn

```diff
@@ -51,6 +51,39 @@ optional true
 longname save well flows to budget file
 description REPLACE save_flows {'{#1}': 'UZF'}
 
+block options
+name wc_filerecord
+type record wc fileout wcfile
+shape
+reader urword
+tagged true
+optional true
+longname
+description
+
+block options
+name wc
```

Comment:
Why not spell out `water_content` instead of cryptic `wc`?

Reply:
addressed by 03b36d1